tabular/src/autogluon/tabular/trainer/model_presets/presets.py

```diff
@@ -81,6 +83,7 @@
     NNFastAiTabularModel: 'NeuralNetFastAI',
     TabTransformerModel: 'Transformer',
     TextPredictorModel: 'TextNeuralNetwork',
+    ImagePredictorModel: 'ImagePredictor',
```

Comment:
to standardize, should it be: `ImageNeuralNetwork` like for Text?

Reply:
I think rather Text should be TextPredictor. This makes it clear that the model is coming from the other predictors within AutoGluon, rather than something custom and specific to Tabular. In future, I'll also add `TabularPredictorModel` as a model type.

Reply:
@sxjscience Can we standardize this in the future?
I also think `TextPredictor` might be more clear name than `TextNeuralNetwork`.

Reply:
Sounds good to me.

Reply:
Updated to TextPredictor